pareto/utilities/results.py

```diff
@@ -113,6 +114,24 @@ def generate_report(
                     "vb_y_Storage",
                     "vb_y_Treatment",
                     "vb_y_Flow",
+                    "vb_y_MVCselected",
```

Comment:
Please rename this variable to a more generic term appropriate for various surrogate models (MVC, MD). Alternatively, consider introducing a new binary variable specifically for MD